### PR TITLE
rlwrap: add page

### PR DESCRIPTION
**Pull request by sohang3112**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.43

### CODE REVIEW

File: pages/common/rlwrap.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Better prompt completion if prompts contain ANSI colour codes:
+
+`rlwrap --ansi-colour-aware {{command}}`
+
+- Enable filename completion (case sensitive):
+
+`rlwrap --complete-filenames {{command}}`
+
+- Add coloured prompts - one of *black*, *red*, *green*, *yellow*, *blue*, *cyan*, *purple (=magenta)* or *white*, or an ANSI-conformant `colour_spec`. An uppercase colour name (Yellow or YELLOW) gives a bold prompt:
```
[the remark this answers is not in the document]
Comment (sohang3112) [reply pages/common/rlwrap.md]: @sebastiaanspeck I removed italic styling and shortened colored prompt description to this:

> Color prompts: black | red | green | yellow | blue | cyan | purple (=magenta) | white | `colour_spec`. Uppercase colour (Yellow | YELLOW) means bold prompt:

Is it ok now?

[the remark this answers is not in the document]
Comment (sohang3112) [reply pages/common/rlwrap.md]: @sebastiaanspeck Do you mean like this:

Add coloured prompts - use colour name, or an ASCI-conformant colour spec. Use uppercase colour name for bold styling:

`rlwrap --prompt-colour {{black | red | green | yellow | blue| cyan| purple (=magenta) | white | colour_spec}} {{command}}`

[the remark this answers is not in the document]
Comment (sohang3112) [reply pages/common/rlwrap.md]: @sebastiaanspeck Your suggestion looks good, but we should also magenta as an alias of purple.

File: pages/common/rlwrap.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# rlwrap
+
+> Add line editing, persistent history and prompt completion to an REPL command.
+> More information: <https://github.com/hanslub42/rlwrap>.
+
+- Run an REPL command with line editing, persistent history and prompt completion:
```
Comment (vitorhcl) [inline pages/common/rlwrap.md:6]:
```suggestion
- Run a REPL command with line editing, persistent history and prompt completion:
```

File: pages/common/rlwrap.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# rlwrap
+
+> Add line editing, persistent history and prompt completion to an REPL command.
```
Comment (vitorhcl) [inline pages/common/rlwrap.md:3]:
```suggestion
> Add line editing, persistent history and prompt completion to a REPL command.
```

**Review by vitorhcl — CHANGES_REQUESTED** [review]
Thanks for your contribution! I suggested some fixes to minor typos.

File: pages/common/rlwrap.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/rlwrap.md]: Yeah https://github.com/tldr-pages/tldr/pull/11941#discussion_r1438572905, this looks much better.

File: pages/common/rlwrap.md (hunk 2 above)
Comment (sohang3112) [reply pages/common/rlwrap.md]: fixed

File: pages/common/rlwrap.md (hunk 3 above)
Comment (sohang3112) [reply pages/common/rlwrap.md]: fixed

File: pages/common/rlwrap.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Better prompt completion if prompts contain ANSI colour codes:
+
+`rlwrap --ansi-colour-aware {{command}}`
+
+- Enable filename completion (case sensitive):
+
+`rlwrap --complete-filenames {{command}}`
+
+- Add coloured prompts, use colour name, or an ASCI-conformant colour spec. Use uppercase colour name for bold styling:
```
Comment (kbdharun) [inline pages/common/rlwrap.md:22]:
```suggestion
- Add coloured prompts, use colour name, or an ASCI-conformant colour specification. Use an uppercase colour name for bold styling:
```
We generally prefer using full words in descriptions (unless the line crosses the 250 characters limit).

File: pages/common/rlwrap.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sohang3112) [reply pages/common/rlwrap.md]: @sebastiaanspeck I tested it, `rlwrap --prompt-colour=magenta {{command}}` works without any error. Magenta and Purple refer to the same colour in rlwrap. I have committed your suggestion but IMO we should still mention somehow that magenta and purple refer to same colour.

**Review by kbdharun — APPROVED** [review]
LGTM, welcome to tldr and thanks for your contribution.

**Comment by sohang3112** [comment]
> Thanks for your contribution! I suggested some fixes to minor typos.

@vitorhcl I committed your suggestion by directly editing in code (not through Github "Accept suggestion" button). How can I tell Github that I have committed your suggestion?

**Comment by vitorhcl** [comment]
> > Thanks for your contribution! I suggested some fixes to minor typos.
> 
> @vitorhcl I committed your suggestion by directly editing in code (not through Github "Accept suggestion" button). How can I tell Github that I have committed your suggestion?

You could re-request my review. @kbdharun already did that.

Thanks!

**Review by vitorhcl — APPROVED** [review]

**Comment by sohang3112** [comment]
@vitorhcl Hi. Please merge this PR since it has been approved now.

**Pull request merged**